process/block/metablock.go

```diff
@@ -1041,6 +1046,7 @@ func (mp *metaProcessor) createPeerInfo() ([]block.PeerData, error) {
 
 // CreateBlockHeader creates a miniblock header list given a block body
 func (mp *metaProcessor) CreateBlockHeader(bodyHandler data.BodyHandler, round uint64, haveTime func() bool) (data.HeaderHandler, error) {
+	log.Info(fmt.Sprintf("starting create block header in round %d\n", round))
```

Comment:
"started creating" - do we really need this log ?

Reply:
Renamed